frontend/projects/upgrade/src/assets/i18n/en.json

```diff
@@ -354,10 +354,10 @@
   "feature-flags.no-flags-in-table.text": "No feature flags is defined. Create a new feature flag.",
   "feature-flags.delete-flag.message.text": "Type the name of feature flag to confirm deletion:",
   "feature-flags.delete-flag.input-placeholder.text": "Feature flag name",
-  "feature-flags.add-flag.text": "ADD FLAG",
-  "feature-flags.add-flag-modal.name-hint.text": "The name for this feature flag.",
-  "feature-flags.add-flag-modal.key-hint.text": "A unique key used to retrieve this feature flag from the client application.",
-  "feature-flags.add-flag-modal.app-context-hint.text": "The App Context indicates where the experiment will run, known to UpGrade.",
+  "feature-flags.upsert-flag.text": "ADD FLAG",
```

Comment:
can we have separate text for the primary button as @zackcl mentioned for add we will have `create` while for update we might have `save` as button text so maybe not upsert here?

Reply:
this key is not actually used! deleted, problem solved.